### ISSUE TITLE
Concise tag filter

### ISSUE THREAD

**Issue by multimeric** [JuanCouste/overpass-ql-ts#20]
I want to represent `way[highway]`, which is a nice simple expression in Overpass QL.

As far as I can tell, the simplest way to represent the above query is this:
```typescript
api.execJson(s => s.way.byTags(builder => ({
    highway: builder.exists()
})))
```
In terms of line number it's not an issue, but I find the amount of nested functions makes it hard to parse:

My first suggestion is if you could extend the object syntax ("`OverpassQueryTagFitlerObject`") to allow booleans to check for existence:

```typescript
api.execJson(s => s.way.byTags({
    highway: true
}))
```

Another alternative might be to just make the builder object a constant:
```typescript
import { FetchOverpassApi, OverpassOutputVerbosity, OverpassFormat, OverpassState, TagBuilder } from "overpass-ql-ts";

api.execJson(s => s.way.byTags({
    highway: TagBuilder.exists()
}))
```

**Comment by JuanCouste** [comment]
The boolean option seems like a good addition,
in the meantime, you can import OverpassTagFilterBuilderImp or OverpassExistsTagFilterHelper and instance it yourself:

```typescript
const exists = new OverpassExistsTagFilterHelper(false); // negated: false
const not_exists = new OverpassExistsTagFilterHelper(true);  // negated: true
```
